cli/src/main.rs

```diff
@@ -69,22 +69,16 @@ pub struct CompileCommand {
 
     /// Path to output PDF file
     output: Option<PathBuf>,
-}
 
-/// Watches the input file and recompiles on changes
-#[derive(Debug, Clone, Parser)]
-pub struct WatchCommand {
-    /// Path to input Typst file
-    input: PathBuf,
-
-    /// Path to output PDF file
-    output: Option<PathBuf>,
+    /// Opens the output file after compilation using the default PDF viewer
+    #[arg(short = 'O', long = "open")]
```

Comment:
I think a lot of people like shorthand version, but if I were to decide, then no. I think that for such features, `--open` the "long" version is better and clearer because otherwise, `-O` may feel like `-Output` which is undesired.

Reply:
I agree, let's remove it.